src/main/pages/che-7/end-user-guide/assembly_making-a-workspace-portable-using-a-devfile.adoc

```diff
@@ -1,62 +1,33 @@
 ---
 title: Making a workspace portable using a Devfile
-keywords: 
+keywords:
 tags: []
 sidebar: che_7_docs
 permalink: che-7/making-a-workspace-portable-using-a-devfile/
 folder: che-7/end-user-guide
-summary: 
+summary:
 ---
 
 :parent-context-of-making-a-workspace-portable-using-a-devfile: {context}
 
-[id='making-a-workspace-portable-using-a-devfile']
-= Making a workspace portable using a devfile
-
 :context: making-a-workspace-portable-using-a-devfile
+[id='making-a-workspace-portable-using-a-devfile_{context}']
+= Making a workspace portable using a devfile
 
-Eclipse Che workspaces are defined by a *devfile*. A devfile is the definition of everything that is used in developer workspaces.
-
-Che uses devfiles to define:
-
-* projects to clone
-* browser IDE to use
-* preconfigured commands
-* tools that you need
-* application runtime definition
-
-When you create a workspace, Che uses that definition to initiate everything for you and run all the containers for your tools and your application runtimes. Che also mounts filesystem volumes to make your source code available to the workspace.
-
-Devfiles can be versioned with the project source code. When you have to get a workspace to fix an old maintenance branch, the project devfile provides you with a definition of the workspace with the tools and the exact dependencies to start working on the old branch. Use it to easily instantiate workspaces on demand.
-
-Che maintains the devfile up-to-date with the tools you use in the workspace:
+To transfer a configured Che 7 workspace, create and export the devfile of the workspace and load the devfile on a different host to initialize a new instance of the workspace. For detailed instructions on how to create such a devfile, see below.
```

Comment:
```suggestion
To transfer a configured Che workspace, create and export the devfile of the workspace and load the devfile on a different host to initialize a new instance of the workspace. For detailed instructions on how to create such a devfile, see below.
```
@MichalMaler, use a link to a doc chapter instead of **see below**.